FirstProject/css/main.css

```diff
@@ -0,0 +1,154 @@
+
+/*Header section*/
+header {
+    background-image: url(../img/feedback-banner.jpg);
+    width: 100%;
+}
+
+/*Body section*/
+body {
+    overflow-x:hidden;
+}
+
+.left_image {
+    height: 290px;
+}
+
+.right_image {
+    float: right;
+}
+
+.right_description {
+    font-weight: bold;
+    margin-top:50px;
+}
+
+.container-fluid {
```

Comment:
I used bootstrap and this is the reason because I used this class, but I will remove it.